webapi/src/main/scala/org/knora/webapi/slice/resourceinfo/domain/IriConverter.scala

```diff
@@ -11,18 +11,17 @@ import zio.ZLayer
 import zio.macros.accessible
 
 import org.knora.webapi.ApiV2Complex
-import org.knora.webapi.IRI
 import org.knora.webapi.InternalSchema
 import org.knora.webapi.messages.SmartIri
 import org.knora.webapi.messages.StringFormatter
 
 @accessible
 trait IriConverter {
-  def asSmartIri(iri: IRI): Task[SmartIri]
-  def asInternalIri(iri: IRI): Task[InternalIri]           = asSmartIri(iri).mapAttempt(_.toInternalIri)
-  def asInternalSmartIri(iri: IRI): Task[SmartIri]         = asSmartIri(iri).mapAttempt(_.toOntologySchema(InternalSchema))
+  def asSmartIri(iri: String): Task[SmartIri]
+  def asInternalIri(iri: String): Task[InternalIri]        = asSmartIri(iri).mapAttempt(_.toInternalIri)
+  def asInternalSmartIri(iri: String): Task[SmartIri]      = asSmartIri(iri).mapAttempt(_.toOntologySchema(InternalSchema))
   def asInternalSmartIri(iri: InternalIri): Task[SmartIri] = asInternalSmartIri(iri.value)
-  def asExternalIri(iri: InternalIri): Task[IRI] =
+  def asExternalIri(iri: InternalIri): Task[String] =
```

Comment:
Yes: 
*  `IRI` is a type alias for `String` and IMHO this simply is not very helpful.`IRI` suggests more than it actually is, e.g. a certain syntax for the String.
*  My IDE (IntelliJ) is complaining sometimes if this type alias is use. It would still compile and run the code but does visually report a type error in the UI thinking that `IRI` is a `Nothing`.